db/fixtures/server_roles.csv

```diff
@@ -2,6 +2,7 @@ name,description,max_concurrent,external_failover,role_scope
 automate,Automation Engine,0,false,region
 database_operations,Database Operations,0,false,region
 database_owner,Database Owner,1,false,database
+embedded_ansible,Embedded Ansible,1,false,zone
```

Comment:
Correct @chessbyte we can only support 1 active ansible at a time in a region. So this needs to be scoped to `region`